### PR TITLE
Update to sylius 1.10

### PR DESCRIPTION
**Pull request by tbuczen**

### CODE REVIEW

**Review by leszczuu — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
   "description": "Bonus points plugin for Sylius.",
   "license": "MIT",
   "require": {
-    "php": "^7.3",
-    "sylius/sylius": "^1.9.0"
+    "php": "^7.4 || ^8.0",
+    "sylius/sylius": "^1.10.0"
```
Comment (damonsson) [inline composer.json]: I saw lot of plugins which are compatible with 1.10 | 1.9 | 1.8. In this plugin not possible? @tgrochowski

Comment (tbuczen) [reply composer.json]: You're right - those 3 supported versions were added to composer.json - all GH actions tests went ok

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ jobs:
     php: [ 7.4, 8.0 ]
     node: [ 10.x ]
     symfony: [ ^4.4, ^5.2 ]
-    sylius: [ ^1.10.0 ]
+    sylius: [ ^1.8.0, ^1.9.0, ^1.10.0 ]
```
Comment (em411) [inline .github/workflows/build.yml]: All tests will run on the latest Sylius version - 1.10.1

File: composer.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "require": {
     "php": "^7.4 || ^8.0",
-    "sylius/sylius": "^1.10.0"
+    "sylius/sylius": "^1.8.0 || ^1.9.0 || ^1.10.0"
```
Comment (em411) [inline composer.json]: Same as above, correct constraints should look like that: `"sylius/sylius": "~1.8.0 || ~1.9.0 || ~1.10.0"`

File: .github/workflows/build.yml (hunk 2 above)
Comment (christopherhero) [reply .github/workflows/build.yml]: and 1.9
```
            matrix:
                php: ["8.0", "7.4"]
                symfony: ["^4.4", "^5.2"]
                sylius: ["~1.9.0", "~1.10.0"]
                node: ["10.x"]
                mysql: ["8.0"]

                exclude:
                    - sylius: ~1.9.0
                      php: 8.0

                    - sylius: ~1.10.0
                      symfony: 4.4
```

**Pull request closed without merging**